blog/2024-08-08-quorum-queues-in-4.0/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "WORK IN PROGRESS: RabbitMQ 4.0: New Quorum Queue Features"
+tags: ["RabbitMQ 4.0", "Quorum Queues"]
+authors: [kura]
+---
+
+RabbitMQ 4.0 (currently in beta) includes new quorum queue features:
+* message priorities
+* consumer priorities with Single Active Consumer
+* checkpoints
+* ...
+
+<!-- truncate -->
+
+## Message Priorities
+
+Support for message priorities has been probably the most demanded quorum queue feature,
+mostly requested by existing classic mirrored queue users who wanted to migrate to quorum queues
+(remember, [RabbitMQ 4.0 will not support classic queue mirroring](https://www.rabbitmq.com/docs/ha)).
```

Comment:
In the docs, we usually say "support for classic queue mirroring was removed for 4.0" to make it extra clear that this is not a "setting disabled by default".

We also mention that non-replicated classic queues are still around and keep being improved.